<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Interrogazione</class>
 <widget class="QMainWindow" name="Interrogazione">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>714</width>
    <height>505</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>714</width>
    <height>505</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>714</width>
    <height>505</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Interrogazione</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Return">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>450</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Indietro</string>
    </property>
   </widget>
   <widget class="QLabel" name="Avg_hour">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>330</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>    Avg hourly</string>
    </property>
   </widget>
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>621</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="setHere_hour">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>335</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>##</string>
    </property>
   </widget>
   <widget class="QLabel" name="Avg_daily">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>370</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Avg. daily</string>
    </property>
   </widget>
   <widget class="QLabel" name="setHere_daily">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>370</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>##</string>
    </property>
   </widget>
   <widget class="QLabel" name="Avg_week">
    <property name="geometry">
     <rect>
      <x>48</x>
      <y>400</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Avg. for week</string>
    </property>
   </widget>
   <widget class="QLabel" name="setHere_week">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>400</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>##</string>
    </property>
   </widget>
   <widget class="QLabel" name="Avg_month">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>430</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Avg. for month</string>
    </property>
   </widget>
   <widget class="QLabel" name="setHere_month">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>430</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>##</string>
    </property>
   </widget>
   <widget class="QLabel" name="data1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>280</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>##########</string>
    </property>
   </widget>
   <widget class="QLabel" name="data2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>280</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>##########</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Find_user">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Codice Cliente</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Da</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>280</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>A</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Find">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>280</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Cerca</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Reset">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>280</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>310</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Tot</string>
    </property>
   </widget>
   <widget class="QLabel" name="setHere_tot">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>310</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>##</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
